cmd/build.go

```diff
@@ -61,7 +61,13 @@ func newLocalExecuteCommand(config *settings.Config) *cobra.Command {
 			opts.log = logger.NewLogger(config.Debug)
 			opts.cl = client.NewClient(config.Host, config.Endpoint, config.Token)
 		},
-		RunE: func(_ *cobra.Command, _ []string) error {
+		RunE: func(cmd *cobra.Command, flags []string) error {
+			for _, f := range flags {
+				if f == "--help" || f == "-h" {
```

Comment:
I think this might work, but wasn't exactly what I was thinking. 

This might be a little more resilient since it uses the same parsing mechanism as everything else:

```go
func(cmd *cobra.Command, args []string) error {
    if err := cmd.Flags().Parse(); err == pflag.ErrHelp {
        return cmd.Help()
    }
    return runExecute(opts)
}
```

I haven't tried this, but I believe it should work.

Reply:
`Parse` expects one parameter `arguments []string`, so I had to add in `args` to get this to compile:

```go
func(cmd *cobra.Command, args []string) error {
	if err := cmd.Flags().Parse(args); err == pflag.ErrHelp {
		return cmd.Help()
	}
	return runExecute(opts)
},
```

This didn't appear to work, though:

```
$ go run main.go local execute --help
Usage of configFlags:
  -c, --config string   config file (default ".circleci/config.yml")
Error: Unable to parse flags
exit status 255
```

I walked through the above code with a debugger, and `err == pflag.ErrHelp` evaluates to false. Looking at the pflag code, it looks like the return value is set to `pflag.ErrHelp` when `-help` was set but not defined.

I'll write tests against the current code in this branch, but I'm still open to suggestions to how to get this working in a way that's consistent with the parsing used elsewhere!